orttraining/orttraining/training_ops/cuda/nn/layer_norm.cc

```diff
@@ -35,18 +35,19 @@ Status LayerNormGrad<T, U>::ComputeInternal(OpKernelContext* p_op_kernel_context
   typedef typename ToCudaType<U>::MappedType CudaU;
   // Inputs
   const Tensor* Y_grad = p_op_kernel_context->Input<Tensor>(0);
-  const Tensor* X = p_op_kernel_context->Input<Tensor>(1);
+  const Tensor* Y = p_op_kernel_context->Input<Tensor>(1);
   const Tensor* scale = p_op_kernel_context->Input<Tensor>(2);
-  const Tensor* mean = p_op_kernel_context->Input<Tensor>(3);
+  const Tensor* bias = p_op_kernel_context->Input<Tensor>(3);
   const Tensor* inv_std_var = p_op_kernel_context->Input<Tensor>(4);
 
   auto Y_grad_data = reinterpret_cast<const CudaT*>(Y_grad->template Data<T>());
-  auto X_data = reinterpret_cast<const CudaT*>(X->template Data<T>());
+  auto Y_data = reinterpret_cast<const CudaT*>(Y->template Data<T>());
   auto scale_data = reinterpret_cast<const CudaT*>(scale->template Data<T>());
-  auto mean_data = reinterpret_cast<const CudaU*>(mean->template Data<U>());
+  auto bias_data = reinterpret_cast<const CudaT*>(bias->template Data<T>());
   auto inv_std_var_data = reinterpret_cast<const CudaU*>(inv_std_var->template Data<U>());
 
-  const TensorShape& x_shape = X->Shape();
+  const TensorShape& y_shape = Y->Shape();
+  const TensorShape& x_shape = y_shape;
```

Comment:
nit: inconsistent variable names vs. the other ones you used in training_ops/cpu/nn/layer_norm,cc